src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -936,9 +936,11 @@
         {% if default_locale.id_lang != translateField.vars.name %}
           {% set classes = classes ~ ' d-none' %}
         {% endif %}
-        <div class="{{ classes }}" style="flex-grow: 1;">
-          {{ form_widget(translateField) }}
-        </div>
+        {% block translatable_fields_with_dropdown_t %}
```

Comment:
Why introduce this new block? Not sure this modif is necessary

Reply:
sorry left when playing around :( removing.